src/Native/libunwind/src/config.h

```diff
@@ -190,7 +190,7 @@
 #ifdef __cplusplus
 // Used to fit UnwindCursor and Registers_xxx types against unw_context_t /
 // unw_cursor_t sized memory blocks.
-#if defined(_LIBUNWIND_IS_NATIVE_ONLY)
+#if defined(_LIBUNWIND_IS_NATIVE_ONLY) && !defined(__i386__)
```

Comment:
@janvorli, this change is undesired. It was failing assertion at:

https://github.com/dotnet/corert/blob/e81eb736e4467c20aaece04a9db23c10e8567b67/src/Native/libunwind/src/UnwindCursor.hpp#L1172-L1173

Here:

* `A` in `UnwindCursor<A, R>` is `libunwind::LocalAddressSpace`
* `R` is `libunwind::Registers_x86`
* `blk_count<UnwindCursor<A, R>>::count` is 19
* `blk_count< unw_cursor_t >::count` is 23

both of these types are defined by libunwind.

Reply:
It seems it means that libunwind has incorrectly defined the _LIBUNWIND_CURSOR_SIZE. I can see that it was set to 23 by your change https://github.com/dotnet/corert/commit/753c19b595773f4fabbba09ef184385324271d45.

Reply:
Thanks for the pointer (didn't realized that llvm-libunwind is there for long time). I have reverted this change and updated the value to 19. Similar fix was made for ARM: https://github.com/dotnet/corert/pull/7564.

Libunwind sources were updated this year, there is an intermediate commit (now squashed), which indicates the value went from 23->19, then 19->15 and then it was changed back to 23 after the update was completed: https://github.com/dotnet/corert/commit/8157d14c1d82f1dbdb90e654dbc5cacdd74319e7#diff-b4c12eb0bf450740f8f027f3851d108f. However, the squashed history in master branch does not reflect the entire chain.